docs/catalog/sinks/PagerDuty.rst

```diff
@@ -0,0 +1,32 @@
+PagerDuty
+##########
+
+Robusta can send playbooks results to the PagerDuty alerts API.
+
+| To configure PagerDuty, a PagerDuty Integration API Key (string) and a PagerDuty Integration URL (url) is needed.
+
+Configuring the PagerDuty sink
+------------------------------------------------
+
+.. admonition:: values.yaml
+
+    .. code-block:: yaml
+
+        sinks_config:
+            - pagerduty_sink:
+                name: main_pagerduty_sink
+                api_key: <api key> # e.g. f653634653463678fadas43534506
+                url: <url> # e.g.  https://events.pagerduty.com/v2/enqueue/
```

Comment:
Can the URL here ever change? From the PagerDuty [docs](https://developer.pagerduty.com/api-reference/368ae3d938c9e-send-an-event-to-pager-duty) I would assume its constant in which case you can define it as a constant in the source code. No need to expose it here as a parameter.

Reply:
I was not sure because my PagerDuty account recommended me different URLs,
e.g. https://events.pagerduty.com/generic/2010-04-15/create_event.json

Reply:
@taivo123 I think we can make it a constant. The other URL is for the [v1 API](https://developer.pagerduty.com/docs/ZG9jOjExMDI5NTc3-events-api-v1#making-a-request) and PagerDuty [recommends](https://support.pagerduty.com/docs/services-and-integrations#should-i-use-events-api-v1-or-v2) using the v2 API when possible.